utils/common/src/test/java/brooklyn/util/exceptions/ExceptionsTest.java

```diff
@@ -53,6 +53,12 @@ public void test12CollapsePropagatedCompoundConcMod() throws Exception {
         assertContains(e, "ConcurrentModification");
     }
     
+    @Test
+    public void testCollapseTextWhenExceptionMessageEmpty() throws Exception {
```

Comment:
Note this test was failing when I added it, but then rebased and found @ahgittin had fixed it in commit be6fd97ec5b318af02a7b5d663a9c7fe1b365c05